test/csi/simple/Makefile

```diff
@@ -1,16 +1,22 @@
 TARGETS := fib-serial foo memops
```

Comment:
Ok, that's implemented now. By default it links with the null tool but you can do `make foo TOOL=print_tool` now.